org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/managers/WrapperValidatorTest.java

```diff
@@ -47,21 +49,26 @@ public void setProperty() throws Exception {
 	}
 
 	@After
-	public void clearProperty() {
+	public void clearProperty() throws IOException {
 		System.clearProperty("gradle.checksum.cacheDir");
+		WrapperValidator.clear();
```

Comment:
need to also call WrapperValidator.clear() in setProperty(), as previous Gradle tests might have mutated  WrapperValidator's state